config/routes.rb

```diff
@@ -69,10 +69,10 @@
         resource :disability_detail, concerns: :confirmable, only: %i[edit update], path: "/disabilities"
       end
 
-      resource :outcome_date, only: %i[edit update], path: "/outcome-date" do
-        member do
-          get "confirm", to: "outcome_dates#confirm"
-        end
+      resource :outcome_details, only: [], path: "outcome-details" do
```

Comment:
`OutcomeDetails` looks like it's just being used as a shell, would it make sense to just use a namespace instead?

Reply:
I had it as a resource in anticipation of the `:edit` route being added (for recording the actual outcome e.g. passed). But thinking about it, since we don't actually know the exact implementation for that yet, it might be less confusing to have it as a namespace for now 👍